redislock.go

```diff
@@ -131,9 +165,14 @@ func Obtain(ctx context.Context, client RedisClient, key string, ttl time.Durati
 	return New(client).Obtain(ctx, key, ttl, opt)
 }
 
+// Obtain is a short-cut for New(...).ObtainMany(...).
+func ObtainMany(ctx context.Context, client RedisClient, keys []string, ttl time.Duration, opt *Options) (*Lock, error) {
+	return New(client).ObtainMany(ctx, keys, ttl, opt)
+}
```

Comment:
Emm... I doubt it saves a lot of code, I'd better remove.

Doesn't prevent us from merging and then polishing it ourselves.

Reply:
Kept it, use it, like it. But we can remove it if you want, ultimately you decide.